api4/team.go

```diff
@@ -1079,6 +1079,11 @@ func teamExists(c *Context, w http.ResponseWriter, r *http.Request) {
 }
 
 func importTeam(c *Context, w http.ResponseWriter, r *http.Request) {
+	if *c.App.Config().ExperimentalSettings.RestrictSystemAdmin {
```

Comment:
This would apply to on-premise installations too. Are we ok with it?

Reply:
@marianunez thoughts here? I think we've only been using this RestrictSystemAdmin setting for toggling things on and off, rather than checking for cloud?

Reply:
Correct, we have relied on this on other endpoints like locking the system logs. This was created initially for Cloud but in a generic way others can leverage. 1/5 we keep consistency and keep using but I'm ok as well with looking at the cloud license. Thoughts @joram?

Would like to get this merged today before the FF

Reply:
+1 on consistency, will wait for thoughts from Joram :)

Reply:
So I think we're good here? @ethervoid @marianunez

Reply:
As discussed in today's planning, let's do the check on cloud licenses here instead of `RestrictSystemAdmin` as suggested by Mario